pom.xml

```diff
@@ -21,6 +22,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-rest</artifactId>
         </dependency>
+        <dependency>
```

Comment:
you don't need this once you don't persist anything

Reply:
removed